.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
+
+[*]
+end_of_line = crlf
+tab_width = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.cs]
+indent_style = space
+csharp_new_line_before_open_brace = none
+csharp_style_var_for_built_in_types = false
+csharp_style_var_when_type_is_apparent = false
+csharp_style_var_elsewhere = false
+dotnet_style_qualification_for_field = false
+dotnet_style_qualification_for_property = false
+csharp_prefer_braces = true
+
+[*.csproj]
+indent_style = space
+tab_width = 2
+indent_size = 2
```

Comment:
i think this do not belong here and a rogue change. even if you want to submit this change, it deserves it's own PR that should be reviewed by the author of the library.

once you remove this rogue part, i guess @kapetan will be happy to review your null reference exception change.